docs/admin_api/rooms.md

```diff
@@ -290,6 +300,7 @@ The following fields are possible in the JSON response body:
 * `guest_access` - Whether guests can join the room. One of: ["can_join", "forbidden"].
 * `history_visibility` - Who can see the room history. One of: ["invited", "joined", "shared", "world_readable"].
 * `state_events` - Total number of state_events of a room. Complexity of the room.
+* `room_type` - the type of the room. 
```

Comment:
An idea to make it more specific.
```suggestion
* `room_type` - The type of the room. It is `null` for normal rooms and `m.space` for spaces.
  See [specification](https://spec.matrix.org/latest/client-server-api/#types).
```

Reply:
good idea,0 I've updated it. thanks for the suggestion.